eng/pipelines/corefx-base.yml

```
@@ -75,6 +75,10 @@ jobs:
         - _msbuildCommonParameters: ''
         - _archiveTestsParameter: ''
         - _finalFrameworkArg: -framework $(_framework)
+        - _testFilterArg: ''
+
+        - ${{ if ne(parameters.testFilter, '') }}:
+          - _testFilterArg: -testfilter $(testfilter)
```

This should use the parameter value, right? 
```suggestion
          - _testFilterArg: -testfilter ${{ parameters.testFilter }}
```